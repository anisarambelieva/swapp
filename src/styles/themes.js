const themes = {
  light: {
    background: "#FFF",
    header: "#000",
    text: {
      primary: "#4BD5EE",
      secondary: "#4E5B6E",
      description: " #4E5B6E",
    },
    button: {
      primary: {
        bg: "#000",
        text: "#FFE300",
      },
      secondary: {
        bg: "#EFF2F7",
        text: "#4BD5EE",
      },
    },
  },
  dark: {
    background: "#333",
    header: "#333",
    text: {
      primary: "#FFE300",
      secondary: "#4BD5EE",
      description: " #ABB1BA",
    },
    button: {
      primary: {
        bg: "#4BD5EE",
        text: "#FFE300",
      },
      secondary: {
        bg: "#3C4858",
        text: "#4BD5EE",
      },
    },
  },
};

export default themes;
