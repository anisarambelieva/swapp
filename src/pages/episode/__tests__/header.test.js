import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";

import themes from "../../../styles/themes.js";
import EpisodeHeader from "../header.js";

describe("<EpisodeHeader />", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <ThemeProvider theme={themes["light"]}>
        <EpisodeHeader title="Title test" subtitle="Subtitle test" />
      </ThemeProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
