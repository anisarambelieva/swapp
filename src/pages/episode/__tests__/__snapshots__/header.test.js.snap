// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EpisodeHeader /> should match snapshot 1`] = `
.c1 {
  font-size: 30px;
  font-weight: bold;
  color: #4BD5EE;
}

.c2 {
  font-size: 25px;
  font-weight: bold;
  color: #4E5B6E;
}

.c0 {
  background-color: white;
  height: 100%;
  padding-left: 50px;
  padding-top: 55px;
  border-radius: 0px 5px 5px 0px;
}

<div>
  <div
    class="c0"
  >
    <h1
      class="c1"
    >
      Title test
    </h1>
    <h2
      class="c2"
    >
      Subtitle test
    </h2>
  </div>
</div>
`;
